tools/loader/Loader.h

```diff
@@ -37,20 +38,26 @@ class Loader {
   std::string caffe2NetWeightFilename_;
   /// ONNX model file name.
   std::string onnxModelFilename_;
-  /// Execution engine for compiling and running.
-  ExecutionEngine EE_{};
+  /// Host Manager for running the model.
+  std::unique_ptr<glow::runtime::HostManager> hostManager_;
+  /// Backend used for saving bundle and quantization.
+  glow::Backend *backend_;
   /// Function containing the model.
   Function *F_{nullptr};
+  /// Module
+  std::unique_ptr<Module> M_;
   /// A map between quantization profiling names of NodeValues that were lowered
   /// from each other. Maps to a set of names of NodeValues and their NodeKinds
   /// that were replaced by the NodeValue (whose output name is the key) that
   /// replaced them.
   LoweredInfoMap loweredMap_;
 
 public:
-  /// Getter for the Function.
+  /// Getter for the Function. This should not be called after compile since the
```

Comment:
Is this still true since strip is now only stripping payloads off Constants?

Reply:
The partitioner can still remove the original function. If profiling or no partitioning happens then it's safe.